compiler/src/lnntoamm/Fn.ts

```diff
@@ -227,11 +209,15 @@ export class OpcodeFn extends Fn {
     __opcodes.put(name, [this]);
   }
 
-  inline(amm: Output, args: Ref[], assign: string, kind: 'const' | 'let' | '') {
+  asHandler(_amm: Output, _event: string) {
+    TODO('opcodes as event listener???');
```

Comment:
I guess any opcode that returns a void type could work, but the zero or one input arguments requirement for handlers means that likely isn't ever possible without a wrapper function?

Reply:
yeah, which is why I feel weird adding support for them... it's not possible right now but if it ever does become possible we should figure out if it's a scenario we want to handle or not